python/ray/tests/test_metrics_export.py

```diff
@@ -20,6 +21,11 @@ def _setup_client_and_exporter(controller):
     return exporter, client
 
 
+"""
+Test Exporter
+"""
+
+
 @patch("ray.dashboard.dashboard.DashboardController")
```

Comment:
hey a high level question. do you think these are testing enough? how about construct a (buggy?) IngestServer flask server and connect our client to it. Just a thought.